monitoring/metricsreporter/metrics_reporter.go

```diff
@@ -137,13 +139,21 @@ var (
 		Name: "ssv_message_non_committee",
 		Help: "The amount of messages not in committee",
 	}, []string{"ssv_msg_type", "decided"})
+	messagesReceivedFromPeer = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "ssv_messages_received_from_peer",
+		Help: "The amount of messages received from the specific peer",
+	}, []string{"peer_id"})
```

Comment:
Recently we merged a PR to reduce the metrics size (#1184) by removing high cardinality labels such as validator public keys.

I think `peer_id` is a similar issue, as it can potentially have an infinite number of unique values over time as we discover new peers. Unless you think we need to query this by peer, and it's possible to remove a peer from this metric after disconnecting from it, I suggest to remove the `peer_id` label.